### ISSUE TITLE
Completions not working

### ISSUE THREAD

**Issue by robertmylne** [laravel-idea/plugin#484]
### Bug description

All features except column completions are working.

![image](https://user-images.githubusercontent.com/10385217/148335597-ab309b79-f3d7-46db-bfcd-e83e3797b30d.png)

![image](https://user-images.githubusercontent.com/10385217/148335558-3ecc9453-c921-4475-bb4a-b56ca0aba598.png)

### Plugin version

Latest

### Operating system

Windows

### Steps to reproduce

Installed as per instructions in existing Laravel application.

**Comment by adelf** [comment]
As I see `password` column is completed, but `password_reset_token` doesn't. Have you added it by migration? Have you run "Generate Eloquent Helper Code" after creating it?

**Comment by robertmylne** [comment]
Yes, I also ran the `Generate Eloquent Helper Code` and yes `password_reset_token` is in the migration file.

It isn't just the `password_reset_token` column it is all columns that have that error. It seems it can't pick up any columns at all.

**Comment by robertmylne** [comment]
I fixed this issue by connecting to my database in PHPStorm `https://www.jetbrains.com/help/phpstorm/mysql.html`. 

This step might need to be added to the Laravel Idea docs `https://laravel-idea.com/docs/eloquent`

**Comment by gorbunov** [comment]
Outdated and solved issue
